quickfixj-dictgenerator/src/main/java/org/quickfixj/dictgenerator/Repository.java

```diff
@@ -54,7 +54,7 @@ public Repository(File repositoryFile) throws Exception {
             throw new Exception("Invalid repository: Missing required files: " + requiredFiles);
         }
 
-        SAXReader reader = new SAXReader();
+        SAXReader reader = SAXReader.createDefault();
```

Comment:
Use `SAXReader.createDefault()` as noted in https://github.com/dom4j/dom4j/releases/tag/version-2.1.3